packages/breadboard/package.json

```diff
@@ -21,7 +21,9 @@
     "build:tsc": "wireit",
     "build:rollup": "wireit",
     "lint": "wireit",
-    "merm": "wireit"
+    "merm": "wireit",
+    "test:ava": "wireit",
+    "test:node": "node --test --loader ts-node/esm tests/**/*.node.test.ts"
```

Comment:
I think we should just use `tsc` instead of `ts-node` pretty much everywhere, including tests. Reasons:

1. We distribute the output of `tsc`, not `ts-node`. Testing the actual `.js` files we distribute will give us more confidence that the code will work for our users too. One can imagine scenarios where `ts-node` works, but `tsc; node` doesn't (one example that comes to mind: what if we accidentally set `emitDeclarationOnly` so we never even build the `.js` files?).

2. Fewer tools/deps are better, all else equal. It means fewer things that can break, fewer things for us devs to understand, and faster installs. IMO the main reasons to use tools like ts-node is "it's annoying to have to remember to run tsc first", but since wireit guarantees that `tsc` runs first, I think the value there is much less.

Reply:
thoughts on this then?
https://github.com/breadboard-ai/breadboard/pull/587/files#diff-e69c85ffe414cd843a42bba4c0c99234a10668a7719c9830f815d85410c09178R121-R125

Reply:
> thoughts on this then? https://github.com/breadboard-ai/breadboard/pull/587/files#diff-e69c85ffe414cd843a42bba4c0c99234a10668a7719c9830f815d85410c09178R121-R125

Yep, that looks good!

One small comment: in wireit, if you have a script that doesn't produce any output like a test, it's usually a good idea to add `"output": []`. This allows the script to get cached. Meaning, if the test passed at time A, and then you run it again at time B, as long as none of its transitive inputs have changed, it will just skip it because it knows "oh, this script already succeeded with these exact inputs, no need to actually run it". That can make CI tests much faster. See https://github.com/google/wireit?tab=readme-ov-file#caching. The reason you have to explicitly set `output` rather than just leaving it undefined is that wireit is a bit conservative; we treat undefined output as "I don't know if this script has any output files, so I'm not going to cache it", whereas an explicitly empty array means "I know for sure I have no output files to cache".